KMB/make_KMB_info.py

```diff
@@ -620,44 +658,43 @@ def make_tag_categories(self, cache):
         Populates self.content_cats
 
         :param cache: cache for category existence
+        :param tag: the tag (string) to be mapped to a category
+        :return: bool whether a category was added
         """
+        # avoid similar cat to fallback in make_commonscat_categories
+        if (self.fmis and tag == 'Fornminnen') or \
+                (self.bbr and tag.startswith('Byggnadsminnen')):
+            return False
+
         tag_map = self.kmb_info.mappings['tags']
         country_map = self.kmb_info.mappings['countries']
-        for tag in self.tag:
-            if (self.fmis and tag == 'Fornminnen') or \
-                    (self.bbr and tag.startswith('Byggnadsminnen')):
-                # avoid similar cat to fallback in make_commonscat_categories
-                continue
-
-            if tag in tag_map:
-                cat = None
-                if not self.land or self.land == 'se' and \
-                        tag_map[tag].get('SE'):
-                    cat = tag_map[tag].get('SE')
-
-                    # attempt municipal categorisation
-                    if self.kommunName:
-                        test_cat = tag_map[tag].get('SE').replace(
-                            'Sweden', '{} Municipality'.format(
-                                self.kommunName))
-                        if self.kmb_info.category_exists(test_cat, cache):
-                            self.needs_place_cat = False
-                            cat = test_cat
-                elif self.land.upper() in country_map and \
-                        tag_map[tag].get('base'):
-                    # guess a category
-                    test_cat = tag_map[tag].get('base').format(
-                        country_map(self.land.upper()))
+        if tag in tag_map:
+            cat = None
+            if (not self.land or self.land == 'se') and tag_map[tag].get('SE'):
+                cat = tag_map[tag].get('SE')
+
+                # attempt municipal categorisation
+                if self.kommunName:
+                    test_cat = tag_map[tag].get('SE').replace(
+                        'Sweden', '{} Municipality'.format(self.kommunName))
                     if self.kmb_info.category_exists(test_cat, cache):
                         self.needs_place_cat = False
                         cat = test_cat
-
-                if not cat:
-                    # fallback independent of country
-                    cat = tag_map[tag].get('default')
-
-                if cat:
-                    self.content_cats.add(cat)
+            elif self.land.upper() in country_map and tag_map[tag].get('base'):
```

Comment:
KMB contains images from al over (e.g. [this](http://kmb.raa.se/cocoon/bild/kont-1e15333d86715968473f0f6375761e43428a2d89?form_name=searchresult&form_action=fetch&id=16001000258504&offset=3)). There won't be any in batch 1 but you will have a bunch of them in batch 2.